inc/wpseo-functions.php

```diff
@@ -9,6 +9,20 @@
 	exit();
 }
 
+if ( ! function_exists( 'wp_rest_api_available' ) ) {
+	/**
+	 * Checks if the WP-REST-API with at least version 2.0 is available.
+	 *
+	 * @return bool
+	 */
+	function wp_rest_api_available() {
```

Comment:
I would put this in `WPSEO_Utils` library instead of heading a top-level function.